cpp/modmesh/view/RPythonConsoleDockWidget.cpp

```diff
@@ -112,13 +124,90 @@ void RPythonConsoleDockWidget::executeCommand()
 {
     std::string const code = m_command_edit->toPlainText().toStdString();
     appendPastCommand(code);
-    m_history_edit->insertPlainText(m_command_edit->toPlainText());
-    m_history_edit->insertPlainText("\n");
+    printCommandHistory();
     m_command_edit->setPlainText("");
     m_command_string = "";
     m_current_command_index = static_cast<int>(m_past_command_strings.size());
     auto & interp = modmesh::python::Interpreter::instance();
-    interp.exec_code(code);
+
+    interp.exec_code(m_config["appEnvName"], code);
```

Comment:
run the code with the provided app environment according to the config

consider that in the future, viewer could support switching app environments, which will be helpful